stageleft/src/lib.rs

```diff
@@ -308,7 +308,7 @@ pub trait IntoQuotedOnce<'a, T, Ctx>:
         &'b Ctx,
         &mut String,
         &mut &'static str,
-        &mut TokenStream,
+        &mut &'static str,
```

Comment:
Really ought to swap this for an actual method on the trait, and do impl<F> Trait for F where F: FnOnce(...)

Reply:
So the issue is that we actually have many variations based on whether this is an `FnOnce` or `FnMut`, so we need these duplicates to handle those... But what we _could_ do is create a struct that captures all these inputs.

Reply:
Yeah an arg struct would be great